tests/functional/spec/features/submitted_responses.spec.js

```diff
@@ -0,0 +1,64 @@
+const helpers = require('../../helpers');
+
+const RadioPage = require('../../pages/features/submitted_responses/radio.page.js');
+const NumberBlockPage = require('../../pages/features/submitted_responses/test-number-block.page.js');
+const SummaryPage = require('../../pages/summary.page.js');
+const ThankYouPage = require('../../pages/thank-you.page.js');
+const SubmissionPage = require('../../pages/features/submitted_responses/submission.page.js');
+
+
+describe('Submitted Responses', function() {
+
+  beforeEach(function() {
+    return helpers.openQuestionnaire('test_submitted_responses.json').then(() => {
+        return browser
+          .click(RadioPage.bacon())
+          .click(RadioPage.submit())
+          .setValue(NumberBlockPage.testCurrency(), 123)
+          .click(NumberBlockPage.submit())
+          .click(SummaryPage.submit());
+
+    });
+  });
+
+  describe('Check submitted responses', function() {
+    it('Given I complete to Thank-You page, When I click the submitted answers link, Then I should be able to view my submitted answers', function() {
+        return browser
+          .click(ThankYouPage.viewSubmitted())
+          .getUrl().should.eventually.contain('view-submission')
+          .getText(SubmissionPage.radioAnswer()).should.eventually.contain('Bacon')
+          .getText(SubmissionPage.testCurrency()).should.eventually.contain('123');
+    });
+
+    it('Given I am viewing my submitted answers, When I click refresh after the timeout period, Then I should be routed back to Thank You page', function() {
+        return browser
+          .pause(5000)
+          .refresh()
+          .getUrl().should.eventually.contain(ThankYouPage.pageName)
+    });
+  });
+
+  describe('Try to view submission after timeout', function() {
+    it('Given I complete to Thank-You page, When I click the submitted answers link after the timeout period, Then I should not be able to view my submitted answers', function() {
+        return browser
+          .isVisible(ThankYouPage.viewSubmitted())
+          .pause(5000)
+          .click(ThankYouPage.viewSubmitted())
+          .getUrl().should.eventually.contain(ThankYouPage.pageName)
+          .getSource().should.eventually.contain('You are no longer able to view and print your answers.');
```

Comment:
Can you access this text with a selector? it will make the error message better if so